src/style.css

```diff
@@ -1 +1,28 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(/src/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars__star :last-child {
```

Comment:
The selector `.stars__star :last-child` is likely incorrect because it targets a descendant of `.stars__star` that is the last child, rather than the last `.stars__star` element itself. If you want to target the last `.stars__star` element, the correct selector would be `.stars__star:last-child` without the space.